src/frameworks/opencl/api/types.rs

```diff
@@ -272,6 +272,28 @@ pub static CL_CONTEXT_DEVICES:                           uint = 0x1081;
 pub static CL_CONTEXT_PROPERTIES:                        uint = 0x1082;
 pub static CL_CONTEXT_NUM_DEVICES:                       uint = 0x1083;
 
+enum_from_primitive! {
+    /// OpenCL context info constants.
+    #[derive(PartialEq, Debug, Copy, Clone)]
+    #[repr(C)]
+    pub enum ContextInfoQuery {
+        REFERENCE_COUNT = 0x1080,
```

Comment:
Actually maybe you have a better idea of what to do here? I couldn't use `CL_REFERENCE_COUNT` defined just a few lines above, but it's the same magic number. `rustc` won't let me use the same variable in this definition.